src/ui/app_view.rs

```diff
@@ -128,7 +130,19 @@ pub fn app_view(config: Config) -> impl View {
 				EventPropagation::Continue
 			})
 			.style(|s| s.flex_grow(1.0)),
-		// TODO: add log-out button for manual logging out
+		icon_button(
+			IconButton {
+				icon: String::from(lock_icon),
+				tooltip: String::from("Lock Vault"),
+				tooltip_signals,
+				..IconButton::default()
+			},
+			move |_| {
+				// TODO: zerosize pass hash
```

Comment:
Can call `config.clear_hash()` in the meantime?
This function doesn't need to be `&mut self` either, due to the arc/rwlock.